### PR TITLE
fix #34068 hochu.tv

### LINKED ISSUE #34068: hochu.tv
### Issue URL (Ads)
[NSFW][http://hochu.tv/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fhochu.tv%2F)
### Comment
Username: @MuxauJI71
> 
> --
> Информация ниже сгенерирована автоматически, не удаляйте ее.
> 
> Custom userscripts:
> https://cdn.adguard.com/public/Userscripts/AdguardPopupBlocker/2.5/popupblocker.user.js
> https://cdn.adguard.com/public/Userscripts/AdguardAssistant/4.2/assistant.user.js
> https://cdn.adguard.com/public/Userscripts/Wot/1.0/wot.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 6.4.1814.4903
Browser: | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/17.17134
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (30), Block Location API
Filters: | AdGuard Russian,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/34068
Branding 1xbet not reproduced but here I removed redirect from player when you click on play/pause. Also blocked push-notifications.

### CODE REVIEW

File: AnnoyancesFilter/sections/push-notifications.txt [hunk 1]
```diff
@@ -19,6 +19,7 @@ wallstreet-online.de,dni.ru,polit.info#%#if (window.PushManager) { window.PushMa
 ||pushtora.com^$third-party
 !
 ||install.searchfrit.com^$third-party
+||selectroduced.com^$third-party
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/push-notifications.txt]: It is a fake push notification. Should be moved to adservers.

File: RussianFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -20,6 +20,9 @@ s4galaxy.ru,3dnews.ru,lenovo-smart.ru#%#Object.defineProperty(window, 'SocialMar
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/37252
 windowspro.ru#%#AG_abortInlineScript(/browserDetect/, 'document.createElement');
+! https://github.com/AdguardTeam/AdguardFilters/issues/34068
+cdntvnet.com#%#//scriptlet("prevent-window-open")
+cdntvnet.com#%#window.open = function() { };
```
Comment (BlazDT) [inline RussianFilter/sections/general_extensions.txt]: Was not working for me.

Should work though:
||cdntvnet.com/jquerymin18.js

**Review by BlazDT — CHANGES_REQUESTED** [review]
||youtvonlinefree.com/foradu^

Stealth referrer on http://hochu.tv/fap-tv-compilation.html:
@@||cdntvnet.com/fap-tv-compilation.php$stealth

**Comment by zloyden** [comment]
Added `||youtvonlinefree.com/foradu/`.
For stealth-mode referrer I changed a bit rule because old version worked only on one category.
> Was not working for me.

That's strange. I checked these rules on extension and app - works fine on my side, but I change to your variant. `||cdntvnet.com/jquerymin18.js` is enough for block redirect.

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**